app/helpers/question_view_helper.rb

```diff
@@ -13,6 +13,18 @@ def legend(question, page_header, conditional)
     }
   end
 
+  def example_date_in_tax_year_of(date)
```

Comment:
api of this method is a bit quirky. as `date` is a required argument if you want to call this without a date one would have to write
```ruby
example_date_in_tax_year_of(nil)
```

you probably want people to write
```ruby
example_date_in_tax_year_of
```

so signature might be better with something like the below
```ruby
def example_date_in_tax_year_of(date = nil)

OR

def example_date_in_tax_year_of(date = Time.zone.today)
```

Reply:
just a minor thing for people who want to consume this api

Reply:
It wasn't my intention that anyone would want to call this method without a date to call it with. I onyl had it dealing with nils since it would have a log.startdate passed in which may be nil.
That was also why the method name ended with of, so that it read fairly semantically.
In any case for now this method is be delete since I didn't know about tax year vs collection year and already existsing CollecitonTimeHelper class